tests/mypy_test.py

```diff
@@ -264,7 +264,7 @@ def run_mypy(
             # Stub completion is checked by pyright (--allow-*-defs)
             "--allow-untyped-defs",
             "--allow-incomplete-defs",
-            "--allow-subclassing-any",  # TODO: Do we still need this now that non-types dependencies are allowed? (#5768)
+            "--allow-subclassing-any",  # See #9491
```

Comment:
If #9491 is accepted, then this line will be removed. If not then this links to the discussion/decision.